src/dialog-user/services/dialogData.ts

```diff
@@ -112,7 +112,7 @@ export default class DialogDataService {
       if (field.type === 'DialogFieldCheckBox' && fieldValue === 'f') {
         validation.isValid = false;
         validation.message = __('This field is required');
-      } else if (fieldValue === '') {
+      } else if (_.isEmpty(fieldValue)) {
```

Comment:
@chalettu mind adding a test? thanks

Reply:
@ohadlevy , just added a test for this scenario